### PR TITLE
Update _journals.tab for cell-press-research

### PR DESCRIPTION
**Pull request by POBrien333**
via https://github.com/citation-style-language/styles/pull/6023/files

### CODE REVIEW

File: cell-press-research/_journals.tab [hunk 1]
```diff
@@ -21,3 +21,4 @@ Molecular Therapy	1525-0016	1525-0024	cell-numeric-superscript	numeric	science	b
 Molecular Therapy - Oncolytics	2372-7705	x	cell-numeric-superscript	numeric	science	biology	https://www.cell.com/molecular-therapy-family/home
 Molecular Therapy - Nucleic Acids	2162-2531	x	cell-numeric-superscript	numeric	science	biology	https://www.cell.com/molecular-therapy-family/home
 Molecular Therapy - Methods & Clinical Development	2329-0501	x	cell-numeric-superscript	numeric	science	biology	https://www.cell.com/molecular-therapy-family/home
+Cell Reports Medicine	x	2666-3791	cell-numeric-superscript	numeric	medicine  biology ttps://www.cell.com/cell-reports-medicine/authors
```
Comment (bwiernik) [inline cell-press-research/_journals.tab:24]:
```suggestion
Cell Reports Medicine	x	2666-3791	cell-numeric-superscript	numeric	medicine  biology https://www.cell.com/cell-reports-medicine/authors
```

File: cell-press-research/_journals.tab [hunk 2]
```diff
@@ -21,3 +21,4 @@ Molecular Therapy	1525-0016	1525-0024	cell-numeric-superscript	numeric	science	b
 Molecular Therapy - Oncolytics	2372-7705	x	cell-numeric-superscript	numeric	science	biology	https://www.cell.com/molecular-therapy-family/home
 Molecular Therapy - Nucleic Acids	2162-2531	x	cell-numeric-superscript	numeric	science	biology	https://www.cell.com/molecular-therapy-family/home
 Molecular Therapy - Methods & Clinical Development	2329-0501	x	cell-numeric-superscript	numeric	science	biology	https://www.cell.com/molecular-therapy-family/home
+Cell Reports Medicine	x	2666-3791	cell-numeric-superscript	numeric	medicine  biology https://www.cell.com/cell-reports-medicine/authors
```
Comment (bwiernik) [inline cell-press-research/_journals.tab:24]:
```suggestion
Cell Reports Medicine	x	2666-3791	cell-numeric-superscript	numeric medicine biology https://www.cell.com/cell-reports-medicine/authors
```

File: cell-press-research/_journals.tab [hunk 3]
```diff
@@ -21,4 +21,4 @@ Molecular Therapy	1525-0016	1525-0024	cell-numeric-superscript	numeric	science	b
 Molecular Therapy - Oncolytics	2372-7705	x	cell-numeric-superscript	numeric	science	biology	https://www.cell.com/molecular-therapy-family/home
 Molecular Therapy - Nucleic Acids	2162-2531	x	cell-numeric-superscript	numeric	science	biology	https://www.cell.com/molecular-therapy-family/home
 Molecular Therapy - Methods & Clinical Development	2329-0501	x	cell-numeric-superscript	numeric	science	biology	https://www.cell.com/molecular-therapy-family/home
-Cell Reports Medicine	x	2666-3791	cell-numeric-superscript	numeric	medicine  biology https://www.cell.com/cell-reports-medicine/authors
+Cell Reports Medicine	x	2666-3791	cell-numeric-superscript	numeric medicine biology https://www.cell.com/cell-reports-medicine/authors
```
Comment (bwiernik) [inline cell-press-research/_journals.tab:24]:
```suggestion
Cell Reports Medicine	x	2666-3791	cell-numeric-superscript	numeric	medicine	biology	https://www.cell.com/cell-reports-medicine/authors
```

**Comment by POBrien333** [comment]
Sorry about that. Somehow my tabs are not tabs.

**Comment by bwiernik** [comment]
It looks like your text editor is automatically replacing tabs with 2 spaces. That's the default on Atom and VS Code I believe.

**Pull request merged**